pandas/tests/test_frame.py

```diff
@@ -8881,6 +8881,16 @@ def test_drop(self):
         df.drop(labels=df[df.b>0].index, inplace=True)
         assert_frame_equal(df,expected)
 
+        # multi-index issue
+        # origin of GH 11640
+        df = pd.DataFrame(columns=['a', 'b', 'c', 'd'],
+                          data=[[1, 'b1', 'c1', 3],
+                                [1, 'b2', 'c2', 4]])
+        dg = df.pivot_table(index='a', columns=['b', 'c'], values='d')
+        dg = dg.reset_index()
```

Comment:
I would just change this to a better error message, e.g. you *can* detect that maybe the user needs to pass the ``level`` argument, but this should NOT be automatic.